platforms/browser/www/phonegap/plugin/facebookConnectPlugin/facebookConnectPlugin.js

```diff
@@ -161,16 +161,31 @@ cordova.define("com.phonegap.plugins.facebookconnect.FacebookConnectPlugin", fun
         
         // Bake in the JS SDK
         (function () {
+            // Retrieve the root element to append the script tags to. Use the body if no fb-root was found
+            var root = document.getElementById('fb-root') || document.getElementsByTagName('body')[0];
```

Comment:
Yes, I added this because it wasn't really an option for me to add the fb-root tag in my html code. Then I noticed if not fb-root tag was added, the FB SDK will add it themself. In my opinion it's cleaner to let FB handle the fb-root tag because you don't have to spoil your html code with an empty tag...